### PR TITLE
bump OCP to 4.10.27 and CNV 4.10

### PR DESCRIPTION
**Pull request by stuggi**

### CODE REVIEW

**Comment by stuggi** [comment]
need to fix the OCP version check
```
TASK [fail if OCP version <= 4.6] **********************************************
Thursday 18 August 2022  14:06:30 +0000 (0:00:00.056)       0:00:01.385 ******* 
[0;31mfatal: [localhost]: FAILED! => {[0m
[0;31m    "changed": false[0m
[0;31m}[0m
[0;31m[0m
[0;31mMSG:[0m
[0;31m[0m
[0;31mOCP version 4.1 is not allowed.  Only OCP greater than 4.6 is currently supported for assisted installer deployments.[0m
```

File: ansible/host_prep.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@
     - name: fail if OCP version <= 4.6
       fail:
         msg: "OCP version {{ ocp_version }} is not allowed.  Only OCP greater than 4.6 is currently supported for assisted installer deployments."
-      when: ocp_version <= 4.6
+      when: ocp_version|float <= 4.6
```
Comment (abays) [inline ansible/host_prep.yaml:39]: I'm surprised the `float` filter works here for `4.10`.  It seems to me that `4.10|float` would become `4.1`, which is less than `4.6`.  But if it works, so be it!

Comment (stuggi) [reply ansible/host_prep.yaml:39]: maybe then we need to use `abs`, lets see

Comment (stuggi) [reply ansible/host_prep.yaml]: or just version()

File: ansible/host_prep.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@
     - name: fail if OCP version <= 4.6
       fail:
         msg: "OCP version {{ ocp_version }} is not allowed.  Only OCP greater than 4.6 is currently supported for assisted installer deployments."
-      when: ocp_version <= 4.6
+      when: ocp_version is version('4.6', 'le', strict=True )
```
Comment (abays) [inline ansible/host_prep.yaml:39]: Ah, I did not know that function existed

**Comment by stuggi** [comment]
Image upload fails ... need to debug

```
TASK [Re-Upload base image controller-base-img] ********************************
Friday 19 August 2022  10:30:20 +0000 (0:00:00.457)       0:05:30.417 ********* 
[0;31mfatal: [localhost]: FAILED! => {[0m
[0;31m    "changed": true,[0m
[0;31m    "cmd": "virtctl image-upload dv controller-base-img -n openstack --size=40G --image-path=/opt/http_store/data/images/rhel-guest-image-8.4-992.x86_64.qcow2 --insecure\n",[0m
[0;31m    "delta": "0:05:00.117580",[0m
[0;31m    "end": "2022-08-19 10:35:21.268835",[0m
[0;31m    "rc": 1,[0m
[0;31m    "start": "2022-08-19 10:30:21.151255"[0m
[0;31m}[0m
[0;31m[0m
[0;31mSTDOUT:[0m
[0;31m[0m
[0;31mPVC openstack/controller-base-img not found [0m
[0;31mDataVolume openstack/controller-base-img created[0m
[0;31mWaiting for PVC controller-base-img upload pod to be ready...[0m
[0;31m[0m
[0;31m[0m
[0;31mSTDERR:[0m
[0;31m[0m
[0;31mtimed out waiting for the condition[0m
[0;31m[0m
[0;31m[0m
[0;31mMSG:[0m
[0;31m[0m
[0;31mnon-zero return code[0m
```

**Comment by stuggi** [comment]
> `2022-08-19T10:30:41.735952853Z {"level":"error","ts":1660905041.735828,"logger":"controller.datavolume-controller","msg":"Reconciler error","name":"controller-base-img","namespace":"openstack","error":"no accessMode defined DV nor on StorageProfile for host-nfs-storageclass StorageClass","errorVerbose":"no accessMode defined DV nor on StorageProfile for host-nfs-storageclass StorageClass\nkubevirt.io/containerized-data-importer/pkg/controller.getDefaultVolumeAndAccessMode\n\t/remote-source/app/pkg/controller/util.go:1062\nkubevirt.io/containerized-data-importer/pkg/controller.pvcFromStorage\n\t/remote-source/app/pkg/controller/util.go:989\nkubevirt.io/containerized-data-importer/pkg/controller.RenderPvcSpec\n\t/remote-source/app/pkg/controller/util.go:953\nkubevirt.io/containerized-data-importer/pkg/controller.(*DatavolumeReconciler).Reconcile\n\t/remote-source/app/pkg/controller/datavolume-controller.go:471\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:114\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:311\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:266\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:227\nruntime.goexit\n\t/usr/lib/golang/src/runtime/asm_amd64.s:1581","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:266\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\t/remote-source/app/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:227"}`

for reference https://github.com/kubevirt/containerized-data-importer/blob/main/doc/storageprofile.md

**Pull request merged**